pkg/filter/pipelinerun.go

```diff
@@ -0,0 +1,142 @@
+package filter
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/shipwright-io/build/pkg/apis/build/v1alpha1"
+	"github.com/shipwright-io/triggers/test/stubs"
```

Comment:
Looks like you are co-mingling test code in "production" code. Can you try moving this info (which look like constants?) here or in a separate shared package?

Reply:
Indeed, I exported the constants to a dedicated package, please consider the latest changes.